### ISSUE TITLE
Circos

### ISSUE THREAD

**Issue by kimlu1998** [ubccr/software-layer#408]
I need a Circos module that uses the GCC toolchain versin 13.2.0. Here is the earlier easybuild recipe (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/c/Circos/Circos-0.69-9-GCCcore-11.3.0.eb). Thank you.

**Comment by dsajdak** [comment]
@kimlu1998 which CCR software environment are you using?

**Comment by kimlu1998** [comment]
module-version: ccrsoft/2023.01 default

**Comment by dsajdak** [comment]
@kimlu1998 In `ccrsoft/2023.01` we have GCC 11.2 available, not 13.2.  I can attempt to install circos with the GCC 11.2 toolchain if you'd like.  In the upcoming `ccrsoft/2024.04` software environment we have GCC 13.2 but we're not currently building software for that environment so I don't know when I could have this ready for you if you require GCC 13.2

**Comment by kimlu1998** [comment]
The 11.2 version is fine. If you can install it, that would be great. Thank you.

**Comment by dsajdak** [comment]
Hi @kimlu1998  `circos` has been published to the `ccrsoft\2023.01` software environment:

```
login2:~$ module spider circos

-----------------------------------------------------------------------------------------------------------------------------------------------------------------
  circos: circos/0.69-9
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      Circos is a software package for visualizing data and information. It visualizes data in a circular layout - this makes Circos ideal for exploring
      relationships between objects or positions.

    Properties:
      Visualisation software

    You will need to load all module(s) on any one of the lines below before the "circos/0.69-9" module is available to load.

      gcccore/11.2.0

    Help:

      Description
      ===========
      Circos is a software package for visualizing data and information.
       It visualizes data in a circular layout - this makes Circos ideal for exploring
       relationships between objects or positions.


      More information
      ================
       - Homepage: http://www.circos.ca/





login2:~$ module load gcccore circos
login2:~$ circos --help
Usage:
      # without -conf Circos will search for configuration
      circos

      # use specific configuration file
      circos -conf circos.conf

      # diagnose required modules
      circos -modules

      # detailed debugging for code components
      # see http://www.circos.ca/documentation/tutorials/configuration/debugging
      circos -debug_group GROUP1,[GROUP2,...]

      # full debugging
      circos -debug_group _all

      # absolutely no reporting
      circos ... [-silent]

      # configuration dump of a block (or block tree) of
      # any parameters that match REGEXP (optional)
      circos -cdump [BLOCK1/[BLOCK2/...]]{:REGEXP}
      circos -cdump ideogram
      circos -cdump ideogram:label
      circos -cdump ideogram/spacing

      # override configuration parameters
      circos -param image/radius=2000p -param ideogram/show=no

      # for fun - randomize all colors in the image except for
      # COLOR1, COLOR2,...
      circos -randomcolor COLOR1,[COLOR2,...]
      circos -randomcolor white,black

      # brief help
      circos -h

      # man page
      circos -man

      # version
      circos -v

login2:~$ circos -v
circos | v 0.69-8 | 15 Jun 2019 | Perl 5.034000

```
